src/settings.py

```diff
@@ -13,7 +13,9 @@
     cast=lambda filename: tomllib.loads(Path(filename).read_text()),
     default={},
 )
-LLM_CONFIG = PROJECT_CONFIG.get("llm") or {"temperature": 0.2}
+
+DEFAULT_LLM_CONFIG = {"temperature": 0.2, "model_name": "gpt-3.5-turbo"}
+LLM_CONFIG = {**DEFAULT_LLM_CONFIG, **PROJECT_CONFIG.get("llm", {})}
+MODEL_NAME = LLM_CONFIG.get("model_name", "gpt-3.5-turbo")
```

Comment:
`"gpt-3.5-turbo"` was set in the `DEFAULT_LLM_CONFIG` variable, we can reuse it to avoid two locations in hardcode